### PR TITLE
support pypy

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -7,3 +7,4 @@ pytest
 pytest-cov
 Pygments
 six
+whatthepatch-graingert
```
Comment (adamchainz) [inline requirements.in]: Would prefer to use vanilla whatthepatch, yuo haven't even given @cscorley 24 hours to merge your PR's like cscorley/whatthepatch#13

File: setup.py [hunk 2]
```diff
@@ -42,6 +42,7 @@ def get_version(package):
 
 requirements = [
     'six>=1.9.0',
+    'whatthepatch-graingert==0.0.6',
```
Comment (adamchainz) [inline setup.py]: `>=` and maybe `<1.0.0`

File: requirements.in (hunk 1 above)
Comment (graingert) [inline requirements.in]: "temporarily switch to whatthepatch-graingert"

Comment (graingert) [inline requirements.in]: rebased directly on master, without the patch engine changes.

**Comment by adamchainz** [comment]
So since this doesn't work, can we close it?

**Comment by graingert** [comment]
@adamchainz PyPy have a fix https://bitbucket.org/pypy/pypy/commits/423026acc94c

**Comment by adamchainz** [comment]
Closing due to inactivity

**Pull request closed without merging**